pyccel/codegen/printing/ccode.py

```diff
@@ -412,17 +412,33 @@ def _print_Imag(self, expr):
         else:
             return '0'
 
-    def _print_IsNot(self, expr):
+    def _print_is_operator(self, Op, expr):
```

Comment:
We normally try to avoid starting function names with `_print` if it is not followed by the name of a class, so they don't accidentally catch an instance of something. Could you rename this please

Reply:
You're right, this utility function should not have a name starting with `_print`